packages/devtools-reps/src/object-inspector/utils.js

```diff
@@ -12,10 +13,13 @@ if (typeof window === "object") {
 }
 
 function getValue(item) {
-  let value = get(item, "contents.value", undefined)
-    || get(item, "contents.getterValue", undefined);
+  let value;
 
-  if (!value && nodeHasAccessors(item)) {
+  if (has(item, "contents.value")) {
+    value = get(item, "contents.value");
+  } else if (has(item, "contents.getterValue")) {
+    value = get(item, "contents.getterValue", undefined);
+  } else if (nodeHasAccessors(item)) {
     value = item.contents;
   }
 
```

Comment:
I like this pattern. I personally find early returns nicer than conditional assignments


```js
if (x) {
  return y.x
}
  
if (y) {
  return y.y
}
```